jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,156 +14,170 @@ pipeline {
                 pushd solutions/community-deploy/cloud/AWS
-                    export ROOT_PASSWORD=${ROOT_PASSWORD}              
-                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@"${AWS_IP}" "export ROOT_PASSWORD=$ROOT_PASSWORD && echo $ROOT_PASSWORD | sudo passwd --stdin root && pushd /home/centos/cortx-re/solutions/kubernetes && echo "'"hostname=$HOSTNAME,user=root,pass="'" > hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && cat hosts"
+                    export PRIMARY_PUBLIC_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+                    export WORKER_IP=$(cat ip_public.txt | jq '.[0]','.[1]','.[2]','.[3]' | tr -d '",[]')
+                    export HOST1=$(cat ec2_hostname.txt | jq '.[0]'| tr -d '",[]')
+                    ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"${PRIMARY_PUBLIC_IP}" "sudo -- sh -c 'pushd /home/centos/cortx-re/solutions/kubernetes && ./cluster-setup.sh true && sed -i 's,cortx-docker.colo.seagate.com,${HOST1}:8080,g' /etc/docker/daemon.json && systemctl restart docker && sleep 120'"
+                    for wp in $WORKER_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${wp} "sudo -- sh -c 'sed -i 's,cortx-docker.colo.seagate.com,${HOST1}:8080,g' /etc/docker/daemon.json && systemctl restart docker && sleep 120'";done
                     popd
             '''
             }
         }
-
-        stage ('Setup K8s cluster on EC2') {
+        stage('Generate locally build cortx images') {
             steps {
                 script { build_stage = env.STAGE_NAME }
-                sh label: 'setting up K8s cluster on EC2', script: '''
-                pushd solutions/community-deploy/cloud/AWS                
-                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@"${AWS_IP}" "pushd /home/centos/cortx-re/solutions/kubernetes && sudo ./cluster-setup.sh true"
+                sh label: 'Generate locally build cortx images on worker nodes', script: '''
+                pushd solutions/community-deploy/cloud/AWS
+                    export WORKER_IP=$(cat ip_public.txt | jq '.[0]','.[1]','.[2]','.[3]' | tr -d '",[]')
+                    export HOST1=$(cat ec2_hostname.txt | jq '.[0]'| tr -d '",[]')
+                    export CORTX_SERVER_IMAGE="${HOST1}:8080/seagate/cortx-rgw:2.0.0-0"
+                    export CORTX_DATA_IMAGE="${HOST1}:8080/seagate/cortx-data:2.0.0-0"
+                    export CORTX_CONTROL_IMAGE="${HOST1}:8080/seagate/cortx-control:2.0.0-0"
+                    for wp in $WORKER_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${wp} "sudo docker pull '${CORTX_SERVER_IMAGE}' && sudo docker pull '${CORTX_DATA_IMAGE}' && sudo docker pull '${CORTX_CONTROL_IMAGE}'";done
                     popd
-                '''
+            '''
             }
         }
-
-        stage ('Deploy 1N cortx cluster') {
+        stage('Deploy multi-node cortx cluster') {
```

Comment:
Please change this to `Deploy CORTX cluster`

Reply:
fixed